### PR TITLE
ISSD-3138 [Testray] Implement Components and Teams Filters for Subtasks ListView

### PR DESCRIPTION
**Pull request by claricebrasil**

### CODE REVIEW

File: workspaces/liferay-testray-workspace/client-extensions/liferay-testray-custom-element/src/hooks/useQueryParams.tsx [hunk 1]
```diff
@@ -33,7 +33,7 @@ type Params = {
 	[key: string]: string | number | boolean;
 };
 
-const useQueryParams = () => {
+const useQueryParams = (customFilterFields?: any) => {
```
Comment (rafaelfs98) [inline workspaces/liferay-testray-workspace/client-extensions/liferay-testray-custom-element/src/hooks/useQueryParams.tsx:36]:
```suggestion
const useQueryParams = (customFilterFields?: {projectId: string}) => {
```

**Review by rafaelfs98 — APPROVED** [review]
LGTM :fire:

File: workspaces/liferay-testray-workspace/client-extensions/liferay-testray-custom-element/src/hooks/useQueryParams.tsx (hunk 1 above)
Comment (claricebrasil) [reply workspaces/liferay-testray-workspace/client-extensions/liferay-testray-custom-element/src/hooks/useQueryParams.tsx]: Thank you!

**Comment by claricebrasil** [comment]
ci:test:sf

**Pull request closed without merging**